### PR TITLE
add total price to basket

### PR DESCRIPTION
**Pull request by sbsto**
- added a total price line to the basket

### CODE REVIEW

File: app/javascript/components/Groceries/Groceries.tsx [hunk 1]
```diff
@@ -55,6 +55,7 @@ class Groceries extends React.Component<RouteComponentProps, GroceriesState> {
     this.modifyItemInList = this.modifyItemInList.bind(this);
     this.handleCreateOrder = this.handleCreateOrder.bind(this);
     this.addNewItemToBasket = this.addNewItemToBasket.bind(this);
+    this.cartItemPrice = this.cartItemPrice.bind(this);
```
Comment (AsherWright) [inline app/javascript/components/Groceries/Groceries.tsx]: should be verb (e.g. `getCartTotalPrice`)

File: app/javascript/components/Groceries/Groceries.tsx [hunk 2]
```diff
@@ -236,6 +238,10 @@ class Groceries extends React.Component<RouteComponentProps, GroceriesState> {
       })
     }
   }
+
+  cartItemPrice(cartLineItems: GroceryLineItem[]): number {
+    return cartLineItems.reduce((total, item) => total + item.price * item.quantity, 0);
```
Comment (AsherWright) [inline app/javascript/components/Groceries/Groceries.tsx]: Love this ❤️

File: app/javascript/components/Groceries/components/Basket.tsx [hunk 3]
```diff
@@ -58,4 +60,25 @@ export default class Basket extends React.Component<BasketProps> {
...
+      return (
+        <ListGroup.Item className='font-weight-bold'>
+          <Row>
+            <Col>
+              Total Price:
+            </Col>
+            <Col>
+              ${this.props.totalCartPrice}
+            </Col>
+            <Col></Col>
```
Comment (AsherWright) [inline app/javascript/components/Groceries/components/Basket.tsx]: Why empty cols?

File: app/javascript/components/Groceries/Groceries.tsx (hunk 1 above)
Comment (sbsto) [reply app/javascript/components/Groceries/Groceries.tsx]: done

File: app/javascript/components/Groceries/components/GroceryItem.tsx [hunk 4]
```diff
@@ -45,7 +45,7 @@ export default class GroceryItem extends React.Component<GroceryItemProps> {
           {this.props.name}
         </Col>
         <Col>
-          {this.props.price}
+          ${this.props.price}
```
Comment (AsherWright) [inline app/javascript/components/Groceries/components/GroceryItem.tsx]: In future we will have this in our internationalization work. But for now hardcoding is fine.

File: app/javascript/components/Groceries/components/Basket.tsx (hunk 3 above)
Comment (sbsto) [reply app/javascript/components/Groceries/components/Basket.tsx]: formatting

File: app/javascript/components/Groceries/components/GroceryItem.tsx (hunk 4 above)
Comment (sbsto) [reply app/javascript/components/Groceries/components/GroceryItem.tsx]: +1

**Pull request merged**